server/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "lsp-prisma-server",
```

Comment:
maybe we should give all our packages better names here that make clear this is about a vscode extension etc?

Reply:
so maybe the main one  to prisma-vscode nad the server to prisma-vscode-server and the client to prisma-vscode-client?

Reply:
Yeah, that sounds clear to me.

Is the "client" usually called "client" in this context?

Reply:
yes it is usually referred to as "client" ([see here](https://code.visualstudio.com/api/language-extensions/language-server-extension-guide#adding-additional-language-features)